build/webcodebook.js

```diff
@@ -156,7 +156,7 @@ function init(data) {
 function layout() {
   this.loadingIndicator = this.wrap.append('div').attr('id', 'loading-indicator').style('display', 'none');
   this.title.wrap = this.wrap.append('div').attr('class', 'title section');
-  this.nav.wrap = this.wrap.append('div').attr('class', 'nav section');
+  this.nav.wrap = this.wrap.append('div').attr('class', 'wcb-nav section');
   this.controls.wrap = this.wrap.append('div').attr('class', 'controls section');
```

Comment:
I don't see title or controls in the bootstrap class list, so seems like this is sufficient. If you're feeling paranoid you could prefix those as common css names that might be used on a page that contains a chart, but i'm good with waiting for it to become an issue.

Reply:
Yeah, I vote we run with this and squash future conflicts as they emerge.